src/GitHub.Api/Application/ApplicationConfiguration.cs

```diff
@@ -11,19 +10,13 @@ static ApplicationConfiguration()
         {
             var executingAssembly = typeof(ApplicationConfiguration).Assembly;
             AssemblyName = executingAssembly.GetName();
-            ProductHeader = new ProductHeaderValue(ApplicationInfo.ApplicationSafeName, AssemblyName.Version.ToString());
```

Comment:
Not related directly to this PR, but you should make sure this information is sent along with the requests in the node app, so you might need to set it in the environment as well.

Reply:
Tracked in #617

Reply:
This should be fixed in #599